PopulateTools/gobierto
# Use dynamic translations

Closes [#312](/PopulateTools/issues/issues/312)

## :v: What does this PR do?

This PR fixes how I18n js creates the translations file to be totally dynamic.

## :mag: How should this be manually tested?

1. Check budget indicators page has all the translations
2. Check the deploy works

![screen shot 2018-04-12 at 10 23 22](https://user-images.githubusercontent.com/17616/38665004-8d1becc0-3e3b-11e8-9838-15e6c1d459ff.png)
